src/Elmish/Enzyme/Foreign.purs

```diff
@@ -112,6 +114,18 @@ find selector w = liftEffect $ runEffectFn2 find_ selector w
 parent :: forall m n. MonadEffect m => Wrapper n -> m (Wrapper n)
 parent w = liftEffect $ runEffectFn1 parent_ w
 
+-- | Returns the child nodes of the given `Wrapper`. See
+-- | https://enzymejs.github.io/enzyme/docs/api/ReactWrapper/children.html ffor
+-- | more info.
```

Comment:
Do I understand correctly that given:

```
<div><p>foo</p><p>bar</p></div>
<div><p>baz</p></div>
```

doing `find "div" >> children` will return `<p>foo</p><p>bar</p><p>baz</p>`?

If so, you you add a comment to that effect? Similar to what we have on `parent`.

Reply:
It does — done and added a test to make sure this is the case.